### PR TITLE
CLDR 46 Beta2

### PR DESCRIPTION
**Pull request by robertbastian**

### CODE REVIEW

File: provider/source/tests/data/cldr/cldr-units-full/main/ar/units.json [hunk 1]
```diff
@@ -1920,14 +1920,8 @@
...
           "speed-light-speed": {
-            "gender": "feminine",
-            "displayName": "سرعة الضوء",
-            "unitPattern-count-zero": "سرعة الضوء {0}",
-            "unitPattern-count-one": "سرعة الضوء {0}",
-            "unitPattern-count-two": "سرعة الضوء {0}",
-            "unitPattern-count-few": "سرعة الضوء {0}",
-            "unitPattern-count-many": "سرعة الضوء {0}",
-            "unitPattern-count-other": "سرعة الضوء {0}"
+            "displayName": "light",
```
Comment (robertbastian) [inline provider/source/tests/data/cldr/cldr-units-full/main/ar/units.json:1923]: light speed broken in many languages @sffc @srl295

File: components/timezone/src/windows_tz.rs
Comment (robertbastian) [inline components/timezone/src/windows_tz.rs:1]: @nekevss FYI

**Review by sffc — DISMISSED** [review]
Weird that light speed is broken. Seems like something CLDR should fix before the release

File: provider/source/tests/data/cldr/cldr-units-full/main/ar/units.json (hunk 1 above)
Comment (sffc) [reply provider/source/tests/data/cldr/cldr-units-full/main/ar/units.json:1923]: Confirmed that this is apparently WAI (this is a new item, and CLDR rolled back the localized data which was translated incorrectly across the board and they didn't have time to replace it with new data)

**Review by sffc — APPROVED** [review]

**Comment by macchiati** [comment]
I'll add a note about this to the release page. Examining the data, many of
the translators misunderstood what was required, and so we ended up
dropping the data for some locales.

**Pull request merged**